specification/appcomplianceautomation/AppComplianceAutomation.Management/tspconfig.yaml

```diff
@@ -0,0 +1,28 @@
+emit:
+  - "@azure-tools/typespec-autorest"
+options:
+  "@azure-tools/typespec-autorest":
+    use-read-only-status-schema: true
+    emitter-output-dir: "{project-root}/.."
+    azure-resource-provider-folder: "resource-manager"
+    output-file: "{azure-resource-provider-folder}/{service-name}/{version-status}/{version}/appcomplianceautomation.json"
+    examples-directory: "{project-root}/examples"
+  "@azure-tools/typespec-python":
+    flavor: azure
+    package-dir: "azure-mgmt-appcomplianceautomation"
+    package-name: "{package-dir}"
+  "@azure-tools/typespec-csharp":
+    flavor: azure
+    package-dir: "Microsoft.AppComplianceAutomation"
+    clear-output-folder: true
+    model-namespace: false
+    namespace: "{package-dir}"
+  "@azure-tools/typespec-ts":
```

Comment:
JS failed with no `service-dir` set in config and this would be used as prefix folder name in SDK repo and see example here https://github.com/Azure/azure-rest-api-specs/blob/main/specification/contosowidgetmanager/Contoso.WidgetManager/tspconfig.yaml#L2C4-L2C15.

Reply:
Python failed for lack of "service-dir". Here is example: https://github.com/Azure/azure-rest-api-specs/pull/29080/files#diff-eb92accee49d9c181cc391b6e60071d9b47754f4495fee29938194862ad2234b
![image](https://github.com/Azure/azure-rest-api-specs/assets/70930885/41983d8e-7ac3-48d8-8df2-5b937697a819)

Reply:
@raych1 is it possible to provide tspconfig.yaml template and when they fill in their service name, those files can automatically replace the template args with real service name. I think we have met this kind of issue several times?

Reply:
Hi @MaryGao~
I updated the tsconfig.yaml, the JS failed. Could you help me have a look~

Reply:
@qiaozha , service team should reference the [TSP sample project](https://github.com/Azure/azure-rest-api-specs/blob/main/documentation/typespec-structure-guidelines.md#service-folder-structure) to config SDK emitter in `tsp-config.yaml`